Library/Homebrew/os/mac/keg.rb

```diff
@@ -1,5 +1,6 @@
 class Keg
   def change_dylib_id(id, file)
+    return if file.dylib_id == id
```

Comment:
Yep, we do. `dylib_id` comes from `macho.dylib_id`, which is a string, and `id` comes from `dylib_id_for`, which converts a Pathname into a string.